src/main/resources/assets/spookytime/lang/de_de.json

```diff
@@ -13,6 +14,17 @@
 	"item.spookytime.winged_strawberry_candy": "Geflügeltes Erdbeerebonbon",
 	"item.spookytime.caramel_apple": "Karamelapfel",
 	"item.spookytime.baked_pumpkin_seeds": "Gebackene Kürbiskerne",
+	"item.spookytime.pumpkin_stew": "Kürbissuppe",
+	"item.spookytime.peppermint": "Pfefferminz",
+	"item.spookytime.wooden_club": "Holzschläger",
+	"item.spookytime.stone_club": "Steinschläger",
+	"item.spookytime.iron_club": "Eisenschläger",
+	"item.spookytime.gold_club": "Goldschläger",
+	"item.spookytime.reapers_scythe": "Sense des Sensenmannes",
+	"item.spookytime.spooky_trumpet": "Gespenstische Trompete",
+	"item.spookytime.witch_water_bucket": "Hexewassereimer",
+	"item.spookytime.blood_bucket": "Bluteimer",
```

Comment:
This translation feels off but idk

Reply:
Would 'Schnitter-Sense' or 'Sense des Schnitter' be a better translation? Or perhaps 'Sense des Todes'?

Reply:
Yeah but if I use this translation it's hilarious

Reply:
and I was talking about the blood bucket translation not the scythe one

Reply:
Perhaps 'Eimer mit Blut'?

Reply:
I'll keep it like this for now. If any Germans complain we can change it.